docs/editors/vscode.md

```diff
@@ -132,6 +132,65 @@ As you type, the symbol outline is also visible at the top of the file.
 
 ```
 
+## Running and debugging your code
+
+Metals supports running or debugging tests and main methods via
+[Debug Adapter Protocol](https://microsoft.github.io/debug-adapter-protocol/),
+used to communicate with the editor, and
+[Bloop](https://scalacenter.github.io/bloop/), which is responsible for starting
+the actual process. This means that applications can be run the same as for any
+other language in the natively supported `Run` view.
```

Comment:
```suggestion
Metals supports running and debugging tests and main methods via the 
[Debug Adapter Protocol](https://microsoft.github.io/debug-adapter-protocol/).
This is used to communicate between the editor and debugger. When using Metals 
the debugger is [Bloop](https://scalacenter.github.io/bloop/), which is responsible for starting the actual process. This means that applications can be run the same as for any
other language in the natively supported `Run` view.
```

Reply:
Applied some of the suggestions, but I switched it up a bit still.